network/p2p/p2p.go

```diff
@@ -95,8 +95,10 @@ type p2pNetwork struct {
 
 	backoffConnector *libp2pdiscbackoff.BackoffConnector
 
-	fixedSubnets  []byte
-	activeSubnets []byte
+	// initialSubnets holds subnets on node startup
+	initialSubnets records.Subnets
+	// currentSubnets holds current subnets which depend on current active validators and committees
+	currentSubnets records.Subnets
```

Comment:
this is not really precise because these subnets should not be unsubscribed from even if all validators associated with them are removed

if we don't like 'fixedSubnets', i'd suggest 'stickySubnets' instead, and in that case 'currentSubnets' could be renamed to 'committeeSubnets' to more correctly distinguish it

(assuming this PR will be merged after the Alan fork where there are no more validators subnets, so committee subnets would make sense :)

Reply:
another option: `persistentSubnets`

Reply:
Both `stickySubnets` and `persistentSubnets` are good candidates :)

I'd also expand expand the comment description to include this part moshe mentioned:

> these subnets should not be unsubscribed from even if all validators associated with them are removed

as well as throw in couple other options: `primarySubnets` / `preferredSubnets`